.github/workflows/linting.yml

```diff
@@ -12,13 +15,18 @@ jobs:
           cache: 'pipenv'
       - name: Install bootstrapper dependencies
         run: pipenv install --dev --deploy
-      - run: pipenv run cookiecutter . --no-input -f
+      - run: |
+          config_file="cookiecutter/${{ matrix.fe_type }}_template.yaml"
+          pipenv run cookiecutter . --config-file $config_file --no-input -f
```

Comment:
Most of our checks should run every time against both React and Vue versions of a build.
The matrix strategy makes this easy to do.
Should we have flake8 linting errors where a change formats fine for a React app but break for Vue, we'll now catch those on the PR.